### ISSUE TITLE
Run Grammarly on Paper 2 and LaTeX newline formatting.

### ISSUE THREAD

**Issue by gully** [gully/blase#74]
I like to run all of my papers through grammarly.  The free tier is very good and I always learn things.  
It doesn't technically support $\LaTeX$, but in my experience you can copy-and-paste the entire paper tex into Grammarly and it more-or-less works if you just ignore it flagging the latex markup.

One hiccup is that the paper currently has a newline character every 100 characters or so, apparently for version control purposes.  This new line stategy makes sense in software development, but in my experience is more of a pain for paper writing.  Its major demerit is that it defies Grammarly's expectation of having a contiguous sentence, so Grammarly doesn't work.

I'd recommend moving away from the software-style new line characters.  Even though LaTeX supports this feature for version control diff purposes, any edits from colaborators are going to completely change this harmonious formatting anyways.  Instead I recommend newlines only after paragraphs or possibly at the end of a sentence.

**Comment by Sujay-Shankar** [comment]
Yeah, I did the newline thing to make it easy for editors and for version control, because scrolling for a mile to find the right sentence does add up in productivity loss. What I could do, and what should solve the valid issues you raise while still retaining the
readability/editability I want could be having each sentence be a line instead of the ~100 characters per line. That way, grammarly doesn't break, the formatting theoretically doesn't change because changing a sentence doesn't mess with the lines and adding a sentence just is an insert operation, and we're still able to locate sentences in a reasonable amount of time when editing. How does that sound?
